tailord/src/fancontrol/profile.rs

```diff
@@ -66,12 +66,13 @@ impl FanProfile {
                 51..=255 => (value.temp - 50).saturating_mul(2).min(100),
             };
             if min_speed > value.fan {
+                let invalid_fan_value = value.fan;
                 value.fan = min_speed;
                 tracing::warn!(
-                "Fan speed {}% at {}°C is too low. Falling back to {min_speed}%: `{file_name:?}`",
-                value.fan,
-                value.temp
-            );
+                    "Fan speed {}% at {}°C is too low. Falling back to {min_speed}%: `{file_name:?}`",
```

Comment:
Not sure about this but formatting seemed off. If it was intended I can fix it back.

Reply:
For formatting, you can just run `rustfmt`.
There are checks in CI for it.

Reply:
Sometimes rustfmt just ignores macros because it can't know for sure whether the macro has Rust-like syntax or something entirely different. Anyway, the new formatting looks more correct.